src/analysis/normed_space/normed_group_quotient.lean

```diff
@@ -363,7 +364,8 @@ quotient_add_group.ker_mk  _
 
 /-- The operator norm of the projection is at most `1`. -/
 lemma norm_normed_mk_le (S : add_subgroup M) : ∥S.normed_mk∥ ≤ 1 :=
-normed_group_hom.op_norm_le_bound _ zero_le_one (λ m, by simp [quotient_norm_mk_le])
+normed_group_hom.op_norm_le_bound _ zero_le_one
+  (λ m, by simp only [normed_mk.apply, one_mul, quotient_norm_mk_le])
```

Comment:
I don't like this, seems like adding some simp lemmas broke this proof, so it seems something in the simp set is not confluent here?